keyboards/basekeys/slice/keymaps/default/keymap.c

```diff
@@ -0,0 +1,140 @@
+#include QMK_KEYBOARD_H
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
```

Comment:
This shouldn't be needed.

```suggestion
```

Reply:
3 keymaps were updated with this change.